data_structures/arrays/kth_largest_element.py

```diff
@@ -0,0 +1,74 @@
+"""
+Given an array of integers and an integer k, find the kth largest element in the array
+
+https://stackoverflow.com/questions/251781/how-to-find-the-kth-largest-element-in-an-unsorted-array-of-length-n-in-on
+"""
+
+
+def partition(arr, low, high):
```

Comment:
No type hints.
Can we come up with a more self-documenting name than `arr`?

Reply:
how about "elements"?